src/components/Map.jsx

```diff
@@ -0,0 +1,100 @@
+import {useLoadScript, GoogleMap} from '@react-google-maps/api';
+import React from 'react';
+import {useState, useEffect} from 'react';
+
+import useGPSStore from '../hooks/useGPSStore';
+import useThemeStore from '../hooks/useThemeStore';
+import {DayStyle} from '../mapstyles/DayStyle';
+import {NightStyle} from '../mapstyles/NightStyle';
+
+export default function Map({mapTarget, mapZoom}) {
+	// Zustand ---------------------------------------------------->
+
+	//const isGPSCentered = useGPSStore(state => state.isGPSCentered);
+	const setIsGPSCentered = useGPSStore(state => state.setIsGPSCentered);
+	//--
+	const userGPS = useGPSStore(state => state.userGPS);
+	//const setUserGPS = useGPSStore(state => state.setUserGPS);
+	//--
+	const mapCenter = useGPSStore(state => state.mapCenter);
+	const setMapCenter = useGPSStore(state => state.setMapCenter);
+	//--
+	const isNightMode = useThemeStore(state => state.isNightMode);
+	console.log('Map: ', isNightMode);
+
+	// End of Zustand ----------------------------------------------
+
+	// The things we need to track in state
+	const [mapRef, setMapRef] = useState(null);
+	//const [center, setCenter] = useState({lat: 53.53, lng: 9.99});
+	const center = {lat: 53.53, lng: 9.99};
+	// Vars
+	// -Maps Background colors
+	//const backgrounds = {dayMode: '#a3c7df', nightMode: '#17263c'};
+
+	const containerStyle = {
+		width: '100%',
+		height: '100%',
+	};
```

Comment:
Where does this `const` belongs to?

Reply:
It is an attribute of the @react-google-maps/api to resize the map:
`mapContainerStyle={containerStyle}`
I was struggling a lot with cropped viewport (vH) / cut off zoomControls on mobile devices:

![Screenshot_2022-08-30-14-25-21-499_com android chrome](https://user-images.githubusercontent.com/30600277/187436213-759df855-d3aa-425c-bc32-cad179f76f1d.jpg)

which turned out to be a bug with viewport height on mobile devices https://css-tricks.com/the-trick-to-viewport-units-on-mobile/